src/styles/stopwatch.scss

```diff
@@ -0,0 +1,57 @@
+.stopwatch {
+  display: flex;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+
+  &__seconds {
+    height: $stopwatch-seconds-height;
+    width: $stopwatch-seconds-width;
+    background-color: $stopwatch-seconds-color;
+    position: absolute;
+    bottom: 40vmin;
```

Comment:
The `bottom` property in `.stopwatch__seconds` should be half the size of `$stopwatch-size` to center the hand correctly according to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).